07-workbase/00-overview.md

```diff
@@ -8,7 +8,7 @@ toc: false
 ## What is Workbase?
 The Grakn Workbase, along with the [Graql Console](/docs/running-grakn/console) and the [Grakn Clients](/docs/client-api/overview), is an interface through which we can read from a Grakn knowledge graph.
 Workbase allows us to execute Graql [`get`](/docs/query/get-query) and [`compute path`](/docs/query/compute-query#compute-the-shortest-path) queries, and visualise and investigate their results.
-Whether you need a tool to test and experiment with your newly created Grakn knowledge graph, or that you prefer a graphical interface for reading data from Grakn, you will find Workbase extremely useful.
+Whether we need a tool to test and experiment with our newly created Grakn knowledge graph, or that we prefer a graphical interface for reading data from Grakn, we will find Workbase extremely useful.
```

Comment:
This is actually one of the few places where we intentionally want to use "you" instead of "we" :)